proposals/instructor-training-selection.md

```diff
@@ -0,0 +1,144 @@
+# Instructor Training Selection
+
+**Warning: This is only a draft.**
+
+**Warning: This only applies for the two-day training course in December 2015.**
+
+## Terminology
+
+-   An **instructor** is someone who teaches a workshop. Instructor training (IT)
+    prepares them to do this.
+
+## Why Select Attends for Instructor Training
+
+SCF/DC wants to train as more instructors as possible
+but demand for instructor training is higher than what SCF could offer
+at the moment.
+
+## Eligibility
+
+### Groups
+
```

Comment:
On 06/10/15 17:54, Raniere Silva wrote:
> In proposals/instructor-training-selection.md
> <https://github.com/swcarpentry/board/pull/53#discussion_r41283251>:
>
>> +
>> +## Terminology
>> +
>> +-   An **instructor** is someone who teaches a workshop. Instructor training (IT)
>> +    prepares them to do this.
>> +
>> +## Why Select Attends for Instructor Training
>> +
>> +SCF/DC wants to train as more instructors as possible
>> +but demand for instructor training is higher than what SCF could offer
>> +at the moment.
>> +
>> +## Eligibility
>> +
>> +### Groups
>> +
>
> The new title doesn't make sense to me.

Ah. I made two new suggestions, now that I understood what you meant. 
Either will be good by me.

karin